client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -136,6 +139,12 @@ private synchronized void putBlockId(
   };
 
   public RssShuffleManager(SparkConf conf, boolean isDriver) {
+    try {
+      HadoopAccessorProvider.init(new RssBaseConf());
```

Comment:
RssBaseConf is used for Shuffle Server and Coordinator. We shouldn't use it  in the client.